File: src/main/java/no/nav/service/pensjon/regler/to/VilkarsprovAlderpensjon2011Request.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;

import no.nav.domain.pensjon.regler.beregning2011.AfpLivsvarig;
import no.nav.domain.pensjon.regler.beregning2011.SisteAldersberegning2011;
import no.nav.domain.pensjon.regler.grunnlag.ForholdstallUtvalg;
import no.nav.domain.pensjon.regler.krav.Kravhode;

public class VilkarsprovAlderpensjon2011Request extends ServiceRequest {


	private Kravhode kravhode;
	private Date fom;
	private Date tom;
	private Date afpVirkFom;
	private ForholdstallUtvalg forholdstallUtvalg;
	private AfpLivsvarig afpLivsvarig;
	private SisteAldersberegning2011 sisteBeregning;
	
	private boolean utforVilkarsberegning;



}

File: src/main/kotlin/no/nav/pensjon/regler/to/VilkarsprovAlderpensjon2011Request.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning2011.AfpPrivatLivsvarig
import no.nav.pensjon.regler.domain.beregning2011.SisteAldersberegning2011
import no.nav.pensjon.regler.domain.krav.Kravhode
import java.time.LocalDate
import java.util.*

class VilkarsprovAlderpensjon2011Request : ServiceRequest() {
    var kravhode: Kravhode? = null

    var fomLd: LocalDate? = null

    var tomLd: LocalDate? = null

    var afpVirkFomLd: LocalDate? = null

    var afpPrivatLivsvarig: AfpPrivatLivsvarig? = null
    var sisteBeregning: SisteAldersberegning2011? = null
    var utforVilkarsberegning = false
}